### DISCUSSION TITLE
Fieldtype Picker UI

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#5010] [General]
Hm, as slick as the new fieldtype UI looks )and it does of course), for me it is an improvement as well as a new obstacle. 

Most of the time I don't think about fieldtypes in categories. I know the names of the most fieldtypes and just look for an alphabetical sort order. 

I understand that the new presentation has advantages if you look for new or less used fields or if you are a new user but I would like to have the A.Z sort order back, at least as an option.

For me it is less intuitive as it was before.

**Comment by adevade** [comment]
Maybe there could be a **Common** category at the top, featuring duplicates of the most often used field types.

**Comment by jackmcdade** [comment]
If you know what you’re looking for, why not just type it? The search box is autofocuses so you can type, tab, and enter to select.

**Reply by jensolafkoch** [reply]
Yep, Jack, quite obvious! :-) I just didn't see the search box ..
